packages/access-api/test/helpers/ucanto-test-utils.js

```diff
@@ -1,11 +1,20 @@
 import * as Ucanto from '@ucanto/interface'
 import { Voucher } from '@web3-storage/capabilities'
 import * as assert from 'assert'
+import * as principal from '@ucanto/principal'
+
+/**
+ * @typedef HelperTestContext
+ * @property {Ucanto.Signer<Ucanto.DID<'key'>>} issuer
+ * @property {Ucanto.Signer<Ucanto.DID>} service
+ * @property {Ucanto.ConnectionView<Record<string, any>>} conn
+ * @property {import('miniflare').Miniflare} mf
+ */
```

Comment:
Nit: Can we just stick typedefs like these into TS files, they are so much easier to read.

Reply:
https://github.com/web3-storage/w3protocol/pull/462/commits/2e90305660f9b238c37a2193c09d92dcfe38b450

created a test/helpers/types.ts file we can use for test-related types like this